users/bcat/bcat.h

```diff
@@ -39,7 +39,7 @@ enum user_layer {
 
 /* Custom keycodes shared across keymaps. */
 enum user_keycode {
-    MC_ALTT = SAFE_RANGE,
+    MC_ALTT = 0,
```

Comment:
The `SAFE_RANGE`/`QK_USER` macro is used enumerate user level keycodes *after* core ones. Setting this to zero will cause conflict with their keycodes and core ones.

Reply:
thanks for the explanation.

commited your suggested fix